content-webapp/src/main/webapp/WEB-INF/conf/news/feature-flags-configuration.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ * This file is part of the Meeds project (https://meeds.io/).
+ *
+ * Copyright (C) 2024 Meeds Association contact@meeds.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+-->
+<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_3.xsd http://www.exoplatform.org/xml/ns/kernel_1_3.xsd"
+               xmlns="http://www.exoplatform.org/xml/ns/kernel_1_3.xsd">
+
+    <component>
+        <key>NewsListSettingFeatureProperties</key>
+        <type>org.exoplatform.container.ExtendedPropertyConfigurator</type>
+        <init-params>
+            <properties-param>
+                <name>NewsListSettingFeatureProperties</name>
+                <description>NewsList header setting Translation Feature enablement flag</description>
+                <property name="exo.feature.newsListTranslation.enabled" value="${exo.feature.newsListTranslation.enabled:true}"/>
+            </properties-param>
+        </init-params>
+    </component>
+
+    <external-component-plugins>
```

Comment:
not needed IMO, you can define eXo.env.portal.newsListTranslationEnabled = <%=featureService.isFeatureActiveForUser("newsListTranslation", userName)%>; in the jsp file instead of gtmpl like [here](https://github.com/Meeds-io/social/commit/bfa6198f66a357bafd4c567229c139925bb4744e#diff-d1de1c355a5f68a80e0728af119665aa44b8e6bb26577d2f062be9288c6b1685R64)